tests/test_bgp_af.rb

```diff
@@ -28,7 +28,11 @@ def setup
     super
     # Disable and enable feature bgp before each test to ensure we
     # are starting with a clean slate for each test.
-    config('no feature bgp', 'feature bgp')
+    if platform == :nexus
+      config('no feature bgp', 'feature bgp')
+    elsif platform == :ios_xr
```

Comment:
Personally I think the elsif platform = xyz, could just be replaced with 'else' throughout the code. We only support two platforms currently. No biggy.